dev/docs/builtins-reference.md

```diff
@@ -322,3 +322,11 @@ y = X %*% rand(rows=ncol(X), 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+##outlier-Function
```

Comment:
Thanks a lot, @wizard-420 ! Try to add usage and example and follow the same pattern as used for other functions. :)

Reply:
I need help with how to initialize a boolean data type.

Reply:
I have added the contents. please check if everything is alright